zubhub_frontend/zubhub/src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { useContext } from 'react';
 import { withTranslation } from 'react-i18next';
 import { BrowserRouter as Router, Route, Redirect, Switch } from 'react-router-dom';
-import CreateActivity from './views/create_activity/create_activity';
+import CreateActivity from './views/create_activity/CreateActivity';
```

Comment:
😏

Reply:
Won't conflict. I merged the regression commit into this since it was blocking.